docs/source/hardware-interfaces.rst

```diff
@@ -0,0 +1,34 @@
+.. _hardware_interface_packages:
+
+Hardware Interface Packages
+===========================
+
+The Bluesky library does not provide direct support for communicating with real hardware.
+Instead, we define a high-level abstraction: the :ref:`Bluesky Hardware Interface <hardware_interface>`.
+This allows different experimentalists to use different hardware control systems.
+
+The following packages provide support for real hardware communication from Bluesky:
+
+=============  ========================================================================
+Instrbuilder_  Lightweight package with a focus on SCPI_.
+Ophyd_         Primary hardware interface package for Bluesky with a focus on EPICS_.
```

Comment:
I think this should go first, and the description should probably include the phrase "reference implementation".